src/js/rx/components/SortMenu.jsx

```diff
@@ -21,35 +21,41 @@ class SortMenu extends React.Component {
     this.props.onChange(event.target.value);
   }
 
-  handleClick(sortValue) {
+  handleClick(sortValue, sortOrder) {
     return (event) => {
       event.preventDefault();
-      this.props.onClick(sortValue);
+      this.props.onClick(sortValue, sortOrder);
     };
   }
 
-  render() {
-    const sortBys = this.props.options;
-    const sortLinks = (options) => {
-      return options.map((o, ind) => {
-        // Sets an rx-sort-active class when the option value
-        // matches the selected / default value
-        const selectedClass = classNames({
-          'rx-sort-active': o.value === this.props.selected
-        });
+  renderSortLinks() {
+    const { selected: { value, order } } = this.props;
 
-        return (
-          <li key={ind}>
-            <a
-                className={selectedClass}
-                onClick={this.handleClick(o.value)}>
-              {o.label}
-            </a>
-          </li>
-        );
+    const options = this.props.selected.value === 'lastSubmitDate' ? {
```

Comment:
You've already pulled out `value` in the destructuring above, so could just do `selected.value === ...`.